### PR TITLE
#1505 Mock for organization in RepoCreate

### LINKED ISSUE #1505: Implement the Mock version for RepoCreate with Org
From #1502 

We should also implement the mock version for creating a repo under an organization (``MkGithub.repos().create(...)``)

### PR DESCRIPTION
**Pull request by maviteixeira**
Fixes #1505 

New method to create an organization in RepoCreate.
Added a new test to the organization coordinates.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1519](https://github.com/jcabi/jcabi-github/issues/1519) is now in scope, role is `REV`

File: src/main/java/com/jcabi/github/mock/MkRepos.java [hunk 1]
```diff
@@ -90,8 +90,13 @@ public Github github() {
     public Repo create(
         final RepoCreate settings
     ) throws IOException {
+        String user = this.self;
```
Comment (amihaiemil) [inline src/main/java/com/jcabi/github/mock/MkRepos.java]: @maviteixeira looks good; can you just rename this variable to ``owner``? As in, the ``owner`` is either ``this.self`` (the authenticated user) or an organization.

Comment (maviteixeira) [reply src/main/java/com/jcabi/github/mock/MkRepos.java]: Sure, no problem.

**Comment by amihaiemil** [comment]
@maviteixeira thx, looks great

**Comment by 0crat** [comment]
Job `gh:jcabi/jcabi-github#1519` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#1519](https://github.com/jcabi/jcabi-github/issues/1519) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Pull request merged**